cogeo_mosaic/backends/base.py

```diff
@@ -173,11 +173,13 @@ def assets_for_point(
         self,
         lng: float,
         lat: float,
-        coord_crs: CRS = WGS84_CRS,
+        coord_crs: Optional[CRS] = None,
     ) -> List[str]:
         """Retrieve assets for point."""
         mosaic_tms = self.mosaic_def.tilematrixset or WEB_MERCATOR_TMS
-
+        # default coord_sys should be the mosaic's geographic CRS
+        if not coord_crs:
+            coord_crs = self.tms.rasterio_geographic_crs
```

Comment:
we usually do `coord_crs = coord_crs or self.tms.rasterio_geographic_crs` (it saves one line 😅 )